src/index.ts

```diff
@@ -41,7 +42,14 @@ export async function run(): Promise<void> {
 
     const creds: string = `${username}:${password}`;
 
-    await execute(podman, ['push', '--quiet', '--creds', creds, imageToPush, registryPath]);
+    const args: string[] = ['push', '--quiet', '--creds', creds, imageToPush, registryPath];
+
+    // check if tls-verify is not set to null
+    if (tlsVerify !== '' || tlsVerify !== undefined){
```

Comment:
in JS, `null === undefined` is `false`. 

So, the best way to check for null is `tlsVerify == null` (double equals) which will return `true` if `tlsVerify` is `null` OR `undefined`. 

For a string, you can check not null, not undefined, and not empty all at once with:
`if (tlsVerify)`